src/main/java/de/cyface/collector/verticle/CollectorApiVerticle.java

```diff
@@ -264,7 +264,7 @@ private void setupRoutes(final String publicKey, final String privateKey, final
                 .failureHandler(new AuthenticationFailureHandler());
         // Set up data collector route
         v2ApiRouter.post("/measurements").consumes("multipart/form-data")
-                .handler(BodyHandler.create().setBodyLimit(BYTES_IN_ONE_GIGABYTE).setDeleteUploadedFilesOnEnd(true))
+                .handler(BodyHandler.create().setBodyLimit(BYTES_IN_ONE_GIGABYTE).setDeleteUploadedFilesOnEnd(false))
```

Comment:
Why can't we remove the files after handling the request anymore?

> since collector is not working without

Didn't this work for Vicenza?

Reply:
No it did not. If we remove the file in the current implementation the file is already gone when the upload to the database starts. I need to reimplement this using a streaming solution over the event bus. This PR is only in preparation for the real work.